shared/utils/generateInvoice.ts

```
@@ -103,13 +106,13 @@ export const exportToPDF = async (
           ${invoice.contentData?.buyerInfo?.taxRegistration ? `<p style="font-size: 10px; margin: 3px 0;">VAT: ${invoice.contentData.buyerInfo.taxRegistration}</p>` : ""}
         </div>
       </div>
-      
+
       <div style="margin-bottom: 20px; font-size: 10px;">
-        <strong>Payment Chain:</strong> ${invoice.currencyInfo?.network || "-"}<br>
+        <strong>Payment Chain:</strong> ${paymentCurrencies && paymentCurrencies[0] && paymentCurrencies[0].network ? paymentCurrencies[0].network : "-"}<br>
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)